runtime/src/snapshot_utils.rs

```diff
@@ -52,6 +52,7 @@ const DEFAULT_SNAPSHOT_VERSION: SnapshotVersion = SnapshotVersion::V1_2_0;
 pub(crate) const TMP_BANK_SNAPSHOT_PREFIX: &str = "tmp-bank-snapshot-";
 pub const TMP_SNAPSHOT_ARCHIVE_PREFIX: &str = "tmp-snapshot-archive-";
 pub const DEFAULT_MAX_FULL_SNAPSHOT_ARCHIVES_TO_RETAIN: usize = 2;
+pub const DEFAULT_MAX_INCREMENTAL_SNAPSHOT_ARCHIVES_TO_RETAIN: usize = 100;
```

Comment:
100 seems like a lot! Why not like, say 2?

Reply:
Yeah, 100 is probably too many. I was thinking that the incremental snapshots would be very small. How about a few more, like 4?

Reply:
sgtm. I don't have a major objection to 100, it just seemed like a lot but 🤷🏼

Reply:
Got it. Thanks for the review!